packages/renderer/src/Route.svelte

```diff
@@ -2,12 +2,14 @@
 import { createRouteObject } from 'tinro/dist/tinro_lib';
 import type { TinroBreadcrumb, TinroRouteMeta } from 'tinro';
 import { TelemetryService } from './TelemetryService';
+import { lastPage, currentPage, listPage, detailsPage } from './stores/breadcrumb';
 
 export let path = '/*';
 export let fallback = false;
 export let redirect = false;
 export let firstmatch = false;
 export let breadcrumb = null;
+export let navLevel = undefined;
```

Comment:
navLevel should be a type like `'list' | 'details' | undefined`...

or an enum like `enum NavLevelType { LIST, DETAILS, UNKNOWN}`

Reply:
Tried both, and the type felt more natural and concise. New commit pushed with the change, also renamed to NavigationHint.